### PR TITLE
Patch default scalars plotting edge case

### PR DESCRIPTION
**Pull request by akaszynski**
Ran into an edge case today where a dataset with default plotting scalars will not be overridden with new data. This is due to a check where we see if an array with scalars already exists in the dataset before adding it.

```py
import numpy as np
import pyvista as pv

mesh = pv.Sphere()
mesh['Data'] = np.zeros(mesh.n_points)
mesh.plot(scalars=range(mesh.n_points))
```

### CODE REVIEW

**Review by tkoyama010 — COMMENTED** [review]
LGTM for changes. Do we need to add a test for this?

File: pyvista/__init__.py [hunk 1]
```diff
@@ -73,3 +73,6 @@
 
 # Serialization format to be used when pickling `DataObject`
 PICKLE_FORMAT = 'xml'
+
+# Name used for unnamed scalars
```
Comment (akaszynski) [inline pyvista/__init__.py:77]:
```suggestion
# The name used for unnamed scalars
```

**Review by banesullivan — APPROVED** [review]

**Comment by akaszynski** [comment]
> LGTM for changes. Do we need to add a test for this?

Probably not. It's enough of an edge case that we probably don't have to, but if this crops up again I will.
